endpoints.md

```diff
@@ -40,10 +40,10 @@ Hex.pm uses the following root endpoints:
 
 Hex.pm supports sub-repositories, they can be accessed at the following endpoints, where `REPO` is the repository name:
```

Comment:
I don't think we use the name sub-repository anywhere else - should we keep it as is, or change to e.g.:

> Hex.pm supports private repositories for organizations, they can be (...)